app-backend/app/src/main/scala/toolrun/package.scala

```diff
@@ -0,0 +1,7 @@
+package com.azavea.rf
+
+import com.azavea.rf.datamodel.{PaginatedResponse, ToolRun}
+
+package object toolrun extends RfJsonProtocols {
+    implicit val paginatedToolRunFormat = jsonFormat6(PaginatedResponse[ToolRun])
```

Comment:
I guess obviously it was necessary or there'd be complaints about ambiguous implicits. nevermind